pkg/controller/certificaterequest/certificaterequest_controller.go

```diff
@@ -200,6 +195,15 @@ func (r *ReconcileCertificateRequest) Reconcile(request reconcile.Request) (reco
 		return reconcile.Result{}, err
 	}
 
+	// fetch the clusterdeployment and bail out if there's an outgoing migration annotation again
+	relocating, err = relocationBailOut(r.client, types.NamespacedName{Namespace: request.Namespace, Name: clusterDeploymentName})
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	if relocating {
+		return reconcile.Result{}, nil
```

Comment:
Same suggestion here to update the CR status for better observability.

Reply:
also done